packages/help-text/README.md

```diff
@@ -0,0 +1,328 @@
+<sp-tab value="s">Small</sp-tab>
+<sp-tab-panel value="s">
+
+```html
+<sp-help-text size="s">Passwords must be at least 8 characters.</sp-help-text>
+```
+
+</sp-tab-panel>
+<sp-tab value="m">Medium</sp-tab>
+<sp-tab-panel value="m">
+
+```html
+<sp-help-text size="m">Passwords must be at least 8 characters.</sp-help-text>
+```
+
+</sp-tab-panel>
+<sp-tab value="l">Large</sp-tab>
+<sp-tab-panel value="l">
+
+```html
+<sp-help-text size="l">Passwords must be at least 8 characters.</sp-help-text>
+```
+
+</sp-tab-panel>
+<sp-tab value="xl">Extra Large</sp-tab>
+<sp-tab-panel value="xl">
+
+```html
+<sp-help-text size="xl">Passwords must be at least 8 characters.</sp-help-text>
+```
+
+</sp-tab-panel>
+</sp-tabs>
+
+## Negative
+
+The negative variant of `<sp-help-text>` is used to convey error messages. An error message should be different than the informative message otherwise delivers to the visitor and should show a solution for correcting the error that has been encountered.
+
+```html
+<sp-help-text variant="negative">
+    Create a password with at least 8 characters.
+</sp-help-text>
+```
+
+### Icon
+
+When associated with content that does not supply an icon outlining the presence of an error, use the `icon` attribute to display one as part of the `<sp-help-text>` element.
+
+```html
+<sp-help-text variant="negative" icon>
+    Create a password with at least 8 characters.
+</sp-help-text>
+```
+
+## Disabled
+
+When associated to content the is disabled, use the `disabled` attribute to match the delivery of the `<sp-help-text>` element to that content.
+
+```html
+<sp-help-text disabled>Passwords must be at least 8 characters.</sp-help-text>
+```
+
+## helpTextManager
+
+The `createHelpTextManager()` helper method surfaces a `render()` method and `id` property to be leveraged within elements into which you'd like to be able to slot `<sp-help-text>` elements. The `id` property should be used while in `{ mode: 'internal' }` as part of the `aria-describedby` attribute of the element you would like to describe with your `<sp-help-text>` element(s). The `render(invalid?: boolean)` method accepts the optional `invalid` argument to manage the presence of the `neutral-text` and `negative-text` slots. These slots are stacked within the `help-text` slot, so that content addressed for that slot will always override content addressed to the other two, regardless of their presence.
+
+The `createHelpTextManager()` is called with default options of `{ mode: 'internal' }` and should be leveraged to associate slotted `<sp-help-text>` elements with content within the shadow root of the parent element.
+
+```js
+import { SpectrumElement, html } from '@spectrum-web-components/base';
+import { createHelpTextManager } from '@spectrum-web-components/help-text/manager.js';
+
+export class ParentWithDescribedElement extends SpectrumElement {
+    helpTextManager = createHelpTextManager({ mode: 'internal' });
```

Comment:
I was going to ask why a helper method (`createHelpTextManager`) was needed, instead of just `new HelpTextManager(this)`... but I can't find an implementation for `createHelpTextManager` in this PR. Do I need more coffee?

Reply:
Updated lated 🤦🏼